### PR TITLE
Adding dashboard hardhat plugin docs

### PR DESCRIPTION
**Pull request by kevinbluer**
## PR Description

This PR adds a reference to the Dashboard Hardhat plugin to the existing Truffle Dashboard docs. Preview build available [here](https://bafybeidksm2nl7eoeuha3zvrncilx5y7ni2miwh6ovlgju4jezjwkmb2yq.on.fleek.co/docs/truffle/how-to/use-the-truffle-dashboard/#usage-with-non-truffle-tooling).

### CODE REVIEW

**Review by gnidan — APPROVED** [review]
sweet!

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md [hunk 1]
```diff
@@ -111,9 +111,13 @@ you're using. You can use the Truffle Dashboard with non-Truffle tools such as H
 
 <p class="alert alert-info">
 <i class="far fa-info-circle"></i> <strong>Information</strong>: Truffle Dashboard cannot decode requests
-when using tools other than Truffle; we plan to add support for other tools in the future.
+when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan to add continue adding support for other tools in the future.
 </p>
 
+### Using Truffle Dashboard with the Hardhat plugin
+
+The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seemless integrate with your existing workflow.
```
Comment (cliffoo) [inline src/docs/truffle/how-to/use-the-truffle-dashboard.md:119]:
```suggestion
The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. It extends `hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seamless integration with your existing workflow.
```

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md [hunk 2]
```diff
@@ -111,9 +111,13 @@ you're using. You can use the Truffle Dashboard with non-Truffle tools such as H
 
 <p class="alert alert-info">
 <i class="far fa-info-circle"></i> <strong>Information</strong>: Truffle Dashboard cannot decode requests
-when using tools other than Truffle; we plan to add support for other tools in the future.
+when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan to add continue adding support for other tools in the future.
```
Comment (cliffoo) [inline src/docs/truffle/how-to/use-the-truffle-dashboard.md:114]:
```suggestion
when using tools other than Truffle or Hardhat (with the plugin outlined below); we plan on continue adding support for other tools in the future.
```

**Review by cliffoo — CHANGES_REQUESTED** [review]
Thanks @kevinbluer !

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md (hunk 2 above)
Comment (cds-amal) [reply src/docs/truffle/how-to/use-the-truffle-dashboard.md:114]: still reads odd. drop `continue` in @cliffoo 's suggestion so it's `...plan on adding support ...`

**Review by cds-amal — COMMENTED** [review]
A suggestion on @cliffoo 's suggestion, otherwise looks good.

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md (hunk 2 above)
Comment (fainashalts) [inline src/docs/truffle/how-to/use-the-truffle-dashboard.md:114]: Not to pile on, but here's my suggestion. :) 

```suggestion
when using tools other than Truffle or Hardhat (with the plugin outlined below). We plan to add support for additional tools in the future.
```

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md (hunk 1 above)
Comment (fainashalts) [inline src/docs/truffle/how-to/use-the-truffle-dashboard.md:119]:
```suggestion
The plugin brings the complete Truffle Dashboard experience, including decoded transaction information, to your Hardhat projects. See the [Installation](https://github.com/trufflesuite/truffle/tree/develop/packages/dashboard-hardhat-plugin#installation) steps to get it setup for your project. The Hardhat plugin extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, enabling seamless integrate with your existing workflow.
```

**Comment by kevinbluer** [comment]
Thanks guys (and oops on the typo)...ready for a quick re-review for when someone has a moment 👍

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md (hunk 2 above)
Comment (kevinbluer) [reply src/docs/truffle/how-to/use-the-truffle-dashboard.md]: Awesome, added 👍

File: src/docs/truffle/how-to/use-the-truffle-dashboard.md (hunk 1 above)
Comment (kevinbluer) [reply src/docs/truffle/how-to/use-the-truffle-dashboard.md]: Awesome thanks @fainashalts

**Review by cds-amal — APPROVED** [review]
LGTM

**Pull request merged**